### PR TITLE
Version up

### PR DESCRIPTION
**Pull request by minsulee2**

### CODE REVIEW

File: client/protocol_versions.json [hunk 1]
```diff
@@ -19,5 +19,8 @@
   },
   "0.6.0": {
     "min": "0.5.0"
+  },
+  "0.7.0": {
+    "min": "0.7.0"
```
Comment (liayoo) [inline client/protocol_versions.json:24]: should we set the max of 0.5.x and 0.6.0 to 0.6.0 then? cc: @platfowner

Comment (minsulee2) [reply client/protocol_versions.json:24]: Oh, Good point. I am also waiting for the answer :)

Comment (platfowner) [reply client/protocol_versions.json:24]: Yes, I think so. Thanks!

**Comment by minsulee2** [comment]
PTAL. All addressed!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by minsulee2** [comment]
Thanks thanks!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**